ironfish-cli/src/commands/workers/status.ts

```diff
@@ -23,7 +23,7 @@ export default class Status extends IronfishCommand {
     const { flags } = await this.parse(Status)
 
     if (!flags.follow) {
-      const client = await this.sdk.connectRpc()
+      const client = await this.sdk.connectWalletRpc()
```

Comment:
Same comment as for `rpc:status`: I'm not sure it ever makes sense for this to be a MemoryClient

Reply:
Discussed offline, will be a future improvement since this behavior existed before